systemd/systemd.go

```diff
@@ -395,15 +367,27 @@ func (l Log) Message() string {
 
 // SID is the syslog identifier of the Log, if any; otherwise, "-".
 func (l Log) SID() string {
-	if sid, ok := l["SYSLOG_IDENTIFIER"].(string); ok {
+	if sid, ok := l["SYSLOG_IDENTIFIER"]; ok {
 		return sid
 	}
 
 	return "-"
 }
 
+// PID is the pid of the client pid, if any; otherwise, "-".
+func (l Log) PID() string {
```

Comment:
PID is spelled Pid in e.g. os.ProcessState - maybe we should follow this convention.